### PR TITLE
small ide updates

### PR DESCRIPTION
**Pull request by mirnawong1**
small update per this pr here https://github.com/dbt-labs/cloud-ui/pull/935

approval only, mirna will merge. thank you

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Review by mshaver631 — DISMISSED** [review]

**Review by matthewshaver — DISMISSED** [review]

**Review by thisispvb — DISMISSED** [review]
The status `D` for deleted files exists now as well.

**Comment by mirnawong1** [comment]
> The status `D` for deleted files exists now as well.

Oh nice one thank you! I'm going to update this and merge on Monday 🙏

File: website/docs/docs/get-started/dbt-cloud-features.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ The dbt Cloud IDE comes with features, including better performance and exciting
 
 | Feature  |  Info |
 |---|---|
-| **File state indicators**  |  Ability to see when changes or actions have been made to the file. The indicators **M, U,** and **•** appear to the right of your file or folder name and indicate the actions performed: <br /> <br /> - Unsaved **(•)** &mdash; The IDE detects unsaved changes to your file/folder<br /> - Modification **(M)** &mdash; The IDE detects a modification of existing files/folders<br /> - Untracked **(U)** &mdash; The IDE detects changes made to new files or renamed files 
+| **File state indicators**  |  Ability to see when changes or actions have been made to the file. The indicators **M, A,** and **•** appear to the right of your file or folder name and indicate the actions performed: <br /> <br /> - Unsaved **(•)** &mdash; The IDE detects unsaved changes to your file/folder<br /> - Modification **(M)** &mdash; The IDE detects a modification of existing files/folders<br /> - Added **(A)** &mdash; The IDE detects added files.
```
Comment (mirnawong1) [inline website/docs/docs/get-started/dbt-cloud-features.md]:
```suggestion
| **File state indicators**  |  Ability to see when changes or actions have been made to the file. The indicators **M, D, A,** and **•** appear to the right of your file or folder name and indicate the actions performed: <br /> <br /> - Unsaved **(•)** &mdash; The IDE detects unsaved changes to your file/folder<br /> - Modification **(M)** &mdash; The IDE detects a modification of existing files/folders<br /> - Added **(A)** &mdash; The IDE detects added files<br/> - Deleted **(D)** &mdash; The IDE detects deleted files.
```

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ Now that you've set up your development environment and credentails, you should
...
 | 8. | Change branches and view documentation | Change branches in fewer clicks and focus on developing. You can generate and view your [documentation](/docs/collaborate/build-and-view-your-docs) for your dbt project in real time. You can inspect and verify what your project's documentation will look like before you deploy your changes to production.|
-| 9. | File state indicators | The file state indicators will indicate and track any action or changed made in your project. The indicators **M, U, and  •** appear to the right of your file or folder name, and also under the **Changes** section. |
+| 9. | File state indicators | The file state indicators will indicate and track any action or changed made in your project. The indicators **M, A, and  •** appear to the right of your file or folder name, and also under the **Changes** section. |
```
Comment (mirnawong1) [inline website/docs/docs/get-started/develop-in-the-cloud.md]:
```suggestion
| 9. | File state indicators | The file state indicators will indicate and track any action or change in your project. The indicators **M, D, A, and  •** appear to the right of your file or folder name, and also appear under the **Changes** section. |
```

**Comment by mirnawong1** [comment]
ok made that tweak @thisispvb. If you or @matthewshaver can review and approve, I'd really appreciate it 🙏

**Review by thisispvb — APPROVED** [review]
🎉

**Comment by mirnawong1** [comment]
The addition of A and D file state indicators is now live @bossnunta ! Thanks @thisispvb and @matthewshaver

**Pull request merged**